ansible/roles/java/vars/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+  java_packages:
+    - java-1.8.0-openjdk
```

Comment:
You can override variables in a playbook without creating an additional role.

Reply:
I refactored that out in the next PR. I thought I was going to need more than just this